zio-http/src/main/scala/zio/http/ZClient.scala

```diff
@@ -608,7 +608,7 @@ object ZClient {
       localAddress = None,
       addUserAgentHeader = true,
       webSocketConfig = WebSocketConfig.default,
-      idleTimeout = None,
+      idleTimeout = Some(50.seconds),
```

Comment:
Why `50` and not `60`?

Reply:
Just in case a virtual network or proxy decides to set the idle time out to 60 seconds, we need something even lower than that. On the other hand, it should be as high as possible. 50 is a random compromise I picked.